src/app/component/app-public-nav-bar/app-public-nav-bar.component.ts

```diff
@@ -59,7 +59,7 @@ export class AppPublicNavBarComponent implements OnInit, OnChanges, OnDestroy {
       this.navBar = this.configSvc.pageNavBar
       this.primaryNavbarConfig = this.configSvc.primaryNavBarConfig
     }
-    if ((window.innerWidth < 600) && window.location.href.includes('/public/home')) {
+    if ((window.innerWidth < 600) && (this.configSvc.userProfile === null)) {
```

Comment:
try to check through isXSmall$ observable take a refrence of root.component.ts

Reply:
Done